src/icons/bpmn-data-input-output-association.svg

```diff
@@ -0,0 +1 @@
+<svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><line x1="1.5" y1="30.5" x2="27" y2="5" stroke="#000" stroke-width="2" stroke-linecap="round" stroke-dasharray="1.1,4.3"/><polygon fill="#000" points="31.8033009 0.196699141 26.5 16.1066017 24.9797204 14.5863221 28.7627417 3.2372583 17.4136779 7.02027958 15.8933983 5.5"/></g></svg>
```

Comment:
Maybe a bit pedantic but, I think we should expand the tags here to match the other icons, also makes it easier to read. 
I also think the file name should be renamed to start with bpmn-icon-data-... for the same consistency reasons. 
Aside from that, LGTM 👍

Reply:
Good catch, hadn't noticed. I pushed these changes